src/__tests__/test-utils/use-mutation.util.ts

```diff
@@ -1,19 +1,16 @@
 import { act, renderHook } from '@testing-library/react-hooks'
+import { MutateFunction } from 'react-query'
 
-export default async function executeMutation(callback: any, ...input: any[]) {
-  let mutateToTest: any
-  await act(async () => {
-    const renderHookResult = renderHook(callback)
-    const { result, waitForNextUpdate } = renderHookResult
-    await waitForNextUpdate()
```

Comment:
I'm still unsure how this was working before.  Must be some shenanigans from being wrapped in `act` but there was actually no reason to `waitForNextUpdate` here.